src/eval-frame/components/reps/data-table-rep.jsx

```diff
@@ -47,7 +47,7 @@ CellDetails.propTypes = {
   focusedCol: PropTypes.string
 };
 
-class CellRenderer extends React.Component {
+class CellRenderer extends React.PureComponent {
```

Comment:
i think it does help because IIUC purecomponent does a shallow equality check, and as long as the object passed in is identical by reference, the component will not rerender. but in any case: this PR is just phase 1 of rep cleanup; more to come that will optimize this further, so I'm going to hold off on that refactor as part of this PR

Reply:
sounds good